app/components/IconPicker/utils.ts

```diff
@@ -0,0 +1,35 @@
+export const IconCategory = {
+  All: "All",
+  Frequent: "Frequently Used",
+  Search: "Search Results",
+};
+
+export const FREQUENTLY_USED_COUNT = {
+  Get: 24,
+  Track: 30,
+};
+
+const STORAGE_KEYS = {
+  Base: "icon-state",
+  EmojiSkin: "emoji-skin",
+  IconsFrequency: "icons-freq",
+  EmojisFrequency: "emojis-freq",
+  LastIcon: "last-icon",
+  LastEmoji: "last-emoji",
+};
+
+const getStorageKey = (key: string) => `${STORAGE_KEYS.Base}.${key}`;
+
+export const getEmojiSkinKey = () => getStorageKey(STORAGE_KEYS.EmojiSkin);
```

Comment:
Might as well change all these to be `const` instead of function, avoid recalcluating more than once…

```suggestion
export const emojiSkinToneKey = getStorageKey(STORAGE_KEYS.EmojiSkin);
```